src/main/java/com/wolApi/wolApi/AppSettings.java

```diff
@@ -1,7 +1,19 @@
 package com.wolApi.wolApi;
 
 public class AppSettings {
-    public static String mainVerseURL(String bookIndex, String verseIndex) {
+    private static AppSettings INSTANCE;
+
+    private AppSettings() {}
+
+    public static synchronized AppSettings getInstance() {
```

Comment:
If you're curious as to why I made this `synchronized` was because if we are running in multiple threads/processes in spring boot, this will prevent any race conditions occuring if `INSTANCE` is null